example/main.go

```diff
@@ -0,0 +1,117 @@
+package main
```

Comment:
Why not in cmd?

Reply:
Updated, I'm good with either ways. Previously put under example/ since I think cmd/ may be used for other main launchers.

Reply:
I see, I think it makes sense to keep it in the example then :), sorry.

Reply:
Reverted, no worries.